spec/spec_helper.rb

```diff
@@ -1,3 +1,5 @@
+require 'simplecov'
+SimpleCov.start
```

Comment:
Thanks for adding this! The entire code block to implement SimpleCov looks like this:

```
SimpleCov.start do
  add_filter "spec/rails_helper.rb"
end
```